app/controllers/concerns/two_factor_authenticatable.rb

```diff
@@ -152,7 +152,11 @@ def update_phone_attributes
   def update_idv_state
     now = Time.zone.now
     if idv_context?
-      Idv::Session.new(user_session, current_user).params['phone_confirmed_at'] = now
+      Idv::Session.new(
+        user_session: user_session,
+        current_user: current_user,
+        issuer: session[:sp].try(:[], :issuer)
```

Comment:
I tried to use `#dig` but this is not a regular Hash, it is a Session object which doesn't implement `#dig` currently 😿